docs/misc/CSSModules.md

```diff
@@ -0,0 +1,17 @@
+## CSS modules
+
+### CSS what?
+For a quick breakdown on what CSS modules are, check out [Glen Maddern's post ](http://glenmaddern.com/articles/css-modules) which summarizes pretty well.
+
+### How they're used here
+CSS modules allow for module-scoped styling for components. This is done by `require`ing the styles file and allowing the [Webpack css loader](https://github.com/webpack/css-loader) to process the file as a module. Since there are some styles that need to remain global (mainly webfonts and styles for 3rd party components), we differentiate loaders by naming convention of `.module.less` vs `.less`.
```

Comment:
Nitpick, but in the Webpack loader link, capital CSS please?

And clarify "differentiate loaders" to something more like "differentiate files using the modules option with the webpack CSS loader from those being loaded normally" (feel free to adjust for accuracy).